src/main/resources/reference/human_g1k_v37.json

```diff
@@ -1,393 +1 @@
: 78077248},{"name": "19","length": 59128983},{"name": "20","length": 63025520},{"name": "21","length": 48129895},{"name": "22","length": 51304566},{"name": "X","length": 155270560},{"name": "Y","length": 59373566},{"name": "MT","length": 16569},{"name": "GL000207.1","length": 4262},{"name": "GL000226.1","length": 15008},{"name": "GL000229.1","length": 19913},{"name": "GL000231.1","length": 27386},{"name": "GL000210.1","length": 27682},{"name": "GL000239.1","length": 33824},{"name": "GL000235.1","length": 34474},{"name": "GL000201.1","length": 36148},{"name": "GL000247.1","length": 36422},{"name": "GL000245.1","length": 36651},{"name": "GL000197.1","length": 37175},{"name": "GL000203.1","length": 37498},{"name": "GL000246.1","length": 38154},{"name": "GL000249.1","length": 38502},{"name": "GL000196.1","length": 38914},{"name": "GL000248.1","length": 39786},{"name": "GL000244.1","length": 39929},{"name": "GL000238.1","length": 39939},{"name": "GL000202.1","length": 40103},{"name": "GL000234.1","length": 40531},{"name": "GL000232.1","length": 40652},{"name": "GL000206.1","length": 41001},{"name": "GL000240.1","length": 41933},{"name": "GL000236.1","length": 41934},{"name": "GL000241.1","length": 42152},{"name": "GL000243.1","length": 43341},{"name": "GL000242.1","length": 43523},{"name": "GL000230.1","length": 43691},{"name": "GL000237.1","length": 45867},{"name": "GL000233.1","length": 45941},{"name": "GL000204.1","length": 81310},{"name": "GL000198.1","length": 90085},{"name": "GL000208.1","length": 92689},{"name": "GL000191.1","length": 106433},{"name": "GL000227.1","length": 128374},{"name": "GL000228.1","length": 129120},{"name": "GL000214.1","length": 137718},{"name": "GL000221.1","length": 155397},{"name": "GL000209.1","length": 159169},{"name": "GL000218.1","length": 161147},{"name": "GL000220.1","length": 161802},{"name": "GL000213.1","length": 164239},{"name": "GL000211.1","length": 166566},{"name": "GL000199.1","length": 169874},{"name": "GL000217.1","length": 172149},{"name": "GL000216.1","length": 172294},{"name": "GL000215.1","length": 172545},{"name": "GL000205.1","length": 174588},{"name": "GL000219.1","length": 179198},{"name": "GL000224.1","length": 179693},{"name": "GL000223.1","length": 180455},{"name": "GL000195.1","length": 182896},{"name": "GL000212.1","length": 186858},{"name": "GL000222.1","length": 186861},{"name": "GL000200.1","length": 187035},{"name": "GL000193.1","length": 189789},{"name": "GL000194.1","length": 191469},{"name": "GL000225.1","length": 211173},{"name": "GL000192.1","length": 547496}],"xContigs": ["X"],"yContigs": ["Y"],"mtContigs": ["MT"],"par": [{"start": {"contig": "X","position": 60001},"end": {"contig": "X","position": 2699521}},{"start": {"contig": "X","position": 154931044},"end": {"contig": "X","position": 155260561}},{"start": {"contig": "Y","position": 10001},"end": {"contig": "Y","position": 2649521}},{"start": {"contig": "Y","position": 59034050},"end": {"contig": "Y","position": 59363567}}]}
```

Comment:
That's where I had downloaded the original file with the data. I updated the URL. Should have been: ftp://gsapubftp-anonymous@ftp.broadinstitute.org/bundle/